### PR TITLE
[5.3.x]code formatted.#674

### PR DESCRIPTION
**Pull request by bturamatuyu**
Please review #674.

### CODE REVIEW

File: terasoluna-gfw-common-libraries/terasoluna-gfw-string/src/main/java/org/terasoluna/gfw/common/fullhalf/DefaultFullHalf.java [hunk 1]
```diff
@@ -394,7 +394,7 @@
  * </tr>
  * <tr>
  * <td>｝</td>
- * <td>}</td>
```
Comment (kazuki43zoo) [inline terasoluna-gfw-common-libraries/terasoluna-gfw-string/src/main/java/org/terasoluna/gfw/common/fullhalf/DefaultFullHalf.java]: Please keep current format in this class using `<excludes>` on `pom.xml` .

See http://code.revelc.net/formatter-maven-plugin/examples.html.

**Comment by bturamatuyu** [comment]
@kazuki43zoo san
> Please keep current format in this class using  <excludes>  on  pom.xml  

Fix is completed.

File: terasoluna-gfw-common-libraries/pom.xml [hunk 2]
```diff
@@ -95,6 +95,9 @@
                         <configFile>${project.root.basedir}/../eclipse/formatter.xml
                         </configFile>
                         <lineEnding>CRLF</lineEnding>
+                        <excludes>
+                            <exclude>org/terasoluna/gfw/common/fullhalf/DefaultFullHalf.java</exclude>
```
Comment (kazuki43zoo) [inline terasoluna-gfw-common-libraries/pom.xml]: - [ ] `<exclude>` setting move to `terasoluna-gfw-string/pom.xml`.

**Comment by bturamatuyu** [comment]
@kazuki43zoo san
> <exclude>  setting move to  terasoluna-gfw-string/pom.xml .

Fix is completed.

File: terasoluna-gfw-common-libraries/terasoluna-gfw-string/pom.xml [hunk 3]
```diff
@@ -53,6 +53,11 @@
             <plugin>
                 <groupId>net.revelc.code</groupId>
                 <artifactId>formatter-maven-plugin</artifactId>
+                <configuration>
+                        <excludes>
```
Comment (kazuki43zoo) [inline terasoluna-gfw-common-libraries/terasoluna-gfw-string/pom.xml]: - [ ] Plese fix indent ...

**Comment by bturamatuyu** [comment]
@kazuki43zoo san
>Plese fix indent ...

Fix is completed.

**Comment by kazuki43zoo** [comment]
plz backport!

**Pull request merged**